rspec-tools/rspec_tools/validation/description.py

```diff
@@ -1,55 +1,159 @@
 from bs4 import BeautifulSoup
 from pathlib import Path
 from typing import Final
+from collections import Counter
 
 from rspec_tools.errors import RuleValidationError
 from rspec_tools.rules import LanguageSpecificRule
 from rspec_tools.utils import LANG_TO_SOURCE
 
 import re
 
+def read_file(path):
+  section_names_path = Path(__file__).parent.parent.parent.parent.joinpath(path)
+  return section_names_path.read_text(encoding='utf-8').split('\n')
+
+def parse_names(path):
+  section_names_path = read_file(path)
+  return [s.replace('* ', '').strip() for s in section_names_path if s.strip()]
+
+HOW_TO_FIX_IT = 'How to fix it'
+HOW_TO_FIX_IT_REGEX = re.compile(HOW_TO_FIX_IT)
+
+def parse_education_section_names(path):
+  education_format_file = read_file(path)
+  sections = {}
+  optional_sections = {}
+  current_map = sections
+  current_section_name = ''
+  for line in education_format_file:
+    if line.startswith('== '):
+      section = line.replace('== ', '').strip()
+      if section.endswith('(optional)'):
+        section = section.replace(' (optional)', '')
+        current_map = optional_sections
+      else:
+        current_map = sections
+      if HOW_TO_FIX_IT_REGEX.match(section):
```

Comment:
```suggestion
      if section.starts_with(HOW_TO_FIX_IT):
```
This might be slightly faster, but more importantly it makes it more correlated with the body of the `if`